helpers/Base64.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace oat\tao\helpers;
+
+/**
+ * Class Base64
+ *
+ * @package oat\tao\helpers
+ */
+class Base64
+{
+    /**
+     * @param $data
+     *
+     * @return bool
+     */
+    public static function isEncodedImage($data): bool
+    {
+        return is_string($data) && preg_match('/^(data:.*;base64)/', $data) && getimagesize($data) !== false;
+    }
```

Comment:
Consider adding a less strict `isEncoded()` method, just to complete the helper.

```suggestion
    public function isEncoded($data): bool
    {
        return is_string($data) && preg_match('/^data:.*;base64/', $data)
    }

    /**
     * @param $data
     *
     * @return bool
     */
    public static function isEncodedImage($data): bool
    {
        return self::isEncoded($data) && getimagesize($data) !== false;
    }
```

You don't need a capturing group in the regex BTW.